Comparison Engine/Comparison Engine/User Classes/User.cs

```diff
@@ -17,5 +26,33 @@ public class User
         //Enum defined in UserTrustLogic.cs
         public UserTrustLogic.UserTrustType userTrust { get; set; }
 
+        public void ChangeUsername(string newUsername)
+        {
+            username = newUsername;
+            SaveUserInfo();
+        }
+
+        public void ChangeFaveBar(Bar newBar)
+        {
+            favoriteBar = newBar;
```

Comment:
I think if we want to add a feature to have a favorite bar, then maybe having more than one would make more sense

Reply:
That's definitely fair, will fix that